core/script.runtime/src/kotlin/script/experimental/dependencies/resolvers.kt

```diff
@@ -49,7 +49,7 @@ interface DependenciesResolver : ScriptDependenciesResolver {
 
 data class ScriptReport(val message: String, val severity: Severity = Severity.ERROR, val position: Position? = null) {
     data class Position(val startLine: Int, val startColumn: Int, val endLine: Int? = null, val endColumn: Int? = null)
-    enum class Severity { FATAL, ERROR, WARNING, INFO, DEBUG }
+    enum class Severity { DEBUG, INFO, WARNING, ERROR, FATAL }
```

Comment:
The same comment as above, I'd drop this change unless absolutely necessary.

Reply:
Same as above, I see no reason not to drop it.